givitsite/coordinate/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
+from django import forms
+from friendreq.models import REGION_CHOICES, ITEM_CHOICES , ItemRequest 
 
-# Create your models here.
+
+
+class CoordinatedItems(models.Model):    
+    item = models.CharField(max_length = 40,choices = ITEM_CHOICES)
+    request_id = models.IntegerField(default = 100)
```

Comment:
what is this an id of, and why does it have a default value of 100?

Reply:
It's the id of the related request to which the coordination was made - it will be important once I will need to update the relevant request in 'ItemRequest' DB based on actions made in the coordination UI (open/ close requests). 
didn't really think about the default value I will remove it

Reply:
So there's another table that is currently undefined called itemRequest and this will be a foreign key?  
If so, look here as to how to define a foreign key in django: 
https://docs.djangoproject.com/en/3.1/topics/db/examples/many_to_one/

Reply:
Done

Reply:
I was thinking about using OneToOneField: 
  request_id = models.OneToOneField(
        ItemRequest,
        on_delete=models.CASCADE,
        primary_key=True,
    )
since we plan to have one coordination per request but I rather decide after I actually implement the relationship between the DB's.